app/scripts/views/force_auth.js

```diff
@@ -89,8 +115,12 @@ define(function (require, exports, module) {
       var email = self.relier.get('email');
 
       return self.resetPassword(email)
-        .fail(function (err) {
-          self.displayError(err);
+        .fail(function (error) {
+          if (AuthErrors.is(error, 'UNKNOWN_ACCOUNT')) {
```

Comment:
How should unknown accounts be handled if the user clicks reset password? Should the user be given an opportunity to sign up, if the broker supports it? cc @ryanfeeley